ci-operator/step-registry/hypershift/openstack/e2e/execute/hypershift-openstack-e2e-execute-commands.sh

```diff
@@ -34,6 +46,7 @@ hack/ci-test-e2e.sh \
         --e2e.base-domain="${BASE_DOMAIN}" \
         --e2e.external-dns-domain="service.${BASE_DOMAIN}" \
         --test.run='^TestCreateCluster$' \
+	--test.parallel=1 \
```

Comment:
here I assume that's the number of concurrent `HostedClusters` that can be deployed at the same time.